src/io/CSVLineReader.hpp

```diff
@@ -0,0 +1,33 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+// Copyright The XCSoar Project
+
+#pragma once
+
+#include "LineReader.hpp"
+#include "util/DynamicFifoBuffer.hxx"
+
+#include <vector>
+#include <memory>
+
+/**
+ * Adapter which handles CSV escaped newlines
+ */
+
+class CSVLineReader : public TLineReader {
+  std::unique_ptr<TLineReader> source;
+  std::vector<TCHAR> line;
```

Comment:
`CSVLineReader::line` is most certainly used. See `CSVLineReader::ReadLine()` in _CSVLineReader.cpp_.